pkg/process/checks/net.go

```diff
@@ -285,7 +360,7 @@ func remapDNSStatsByOffset(c *model.Connection, indexToOffset []int32) {
 }
 
 // Connections are split up into a chunks of a configured size conns per message to limit the message size on intake.
-func batchConnections(
+func processConnections(
```

Comment:
i think you should restore the original name, this code path didn't change, and it still does batching